www/ListLibraryItems.js

```diff
@@ -13,15 +13,20 @@ var listLibraryItems = {
 		cordova.exec(onSuccess, onError, 'ListLibraryItems', 'requestReadAuthorization', []);
 	},
 
-	listItems: function(includePictures, includeVideos, includeCloud, onSuccess, onError) {
+	listItems: function(includePictures, includeVideos, includeCloud, mediaBuckets, onSuccess, onError) {
 		console.log('ListLibraryItems plugin: calling listItems...');
```

Comment:
can we add something like `calling listItems for bucket ${mediaBuckets}` ?

Reply:
Fixed